### ISSUE TITLE
NameError: global name 'winappdbg' is not defined

### ISSUE THREAD

**Issue by rchateauneu** [trolldbois/python-haystack#27]
When running this script on Windows 7:

import haystack
from haystack import memory_dumper
memory_dumper.dump(123,"toto.dmp")

... I get this error:

dumper.make_mappings()
  File "C:\Python27\lib\site-packages\haystack-0.34-py2.7.egg\haystack\memory_dumper.py", line 41, in make_mappings
self.dbg = dbg.get_debugger(self._pid)
  File "C:\Python27\lib\site-packages\haystack-0.34-py2.7.egg\haystack\dbg.py", line 244, in get_debugger
    return MyWinAppDebugger(pid)
  File "C:\Python27\lib\site-packages\haystack-0.34-py2.7.egg\haystack\dbg.py", line 78, in __init__
    winappdbg.System.request_debug_privileges()
NameError: global name 'winappdbg' is not defined

It is ok after installing winappdbg with pip.

**Comment by trolldbois** [comment]
yeah. I don't know how to have different requirements per target platform.
Obviously winappdbg is not a requirement on non-windows platform.

**Comment by trolldbois** [comment]
As stated, solution is to install winappdbg with pip

**Comment by trolldbois** [comment]
http://stackoverflow.com/questions/16011379/operating-system-specific-requirements-with-pip
for setup.py
